src/shipit_risk_assessment/VERSION

```diff
@@ -1 +1 @@
-1.0.0
+0.1.0
```

Comment:
I assume you want to bump up the version to ``1.1.0`` since ``1.0.0`` was the starting version.